### PR TITLE
Feat: Paystack integration

### PR DESCRIPTION
**Pull request by ZainabAlayande**

### CODE REVIEW

**Comment by victoradepoju** [comment]
@ZainabAlayande all the new flyway migration versions you created already exist.

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,54 @@
+spring.application.name=hng-java-boilerplate
+
```
Comment (lurldgbodex) [inline src/main/resources/application.properties:2]: do not push your application.properties file

File: src/main/resources/db/migration/V3__update_users_and_related_tables.sql [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
-    ADD COLUMN password VARCHAR(255) NULL,
-    ADD COLUMN user_role VARCHAR(255),
-    ADD COLUMN is_enabled BOOLEAN DEFAULT FALSE,
-    ADD COLUMN created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    ADD COLUMN updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
-
-
+ADD COLUMN password VARCHAR(255) NOT NULL,
+ADD COLUMN user_role VARCHAR(255),
+ADD COLUMN is_enabled BOOLEAN DEFAULT FALSE,
```
Comment (lurldgbodex) [inline src/main/resources/db/migration/V3__update_users_and_related_tables.sql:5]: Do not touch existing migration files

File: src/main/resources/db/migration/V8__add_two_fa_and_secret_key_col.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+-- Add columns to the users table
+ALTER TABLE users
+ADD COLUMN two_FA_status BOOLEAN,
+ADD COLUMN two_FA_secret_key VARCHAR(255);
+
```
Comment (lurldgbodex) [inline src/main/resources/db/migration/V8__add_two_fa_and_secret_key_col.sql:5]: v8 already exists in the db. update your migration version

**Review by lurldgbodex — CHANGES_REQUESTED** [review]
make changes as required

**Review by lurldgbodex — APPROVED** [review]

**Review by Fifetoyi — APPROVED** [review]
lgtm

**Comment by Fifetoyi** [comment]
Add description

**Pull request merged**